README.md

```diff
@@ -15,6 +15,7 @@ apply.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
+| python_install_development_dependencies | A boolean indicating whether or not to install development dependencies, such as the Python headers.  These dependencies are sometimes required, e.g., when one is required to build a wheel for a Python package. | `false` | No |
 | python_install_python2 | A boolean indicating whether or not to install Python 2 alongside Python 3.  Where possible, we also install the necessary dependencies to use Python 2 as Ansible's discovered Python; unfortunately, this is only possible on Amazon Linux 2, Debian Stretch, Debian Buster, and Ubuntu Bionic.  Note also that no matter the value of this variable, Python 2 will not be installed on Debian Bookworm; this is because the corresponding system packages are unavailable for that platform. | `false` | No |
```

Comment:
We don't do this [everywhere yet](https://github.com/search?q=Boolean+user%3Acisagov+language%3AMarkdown&type=code&ref=advsearch), but we should capitalize "Boolean" in our docs:
```suggestion
| python_install_development_dependencies | A Boolean indicating whether or not to install development dependencies, such as the Python headers.  These dependencies are sometimes required, e.g., when one is required to build a wheel for a Python package. | `false` | No |
| python_install_python2 | A Boolean indicating whether or not to install Python 2 alongside Python 3.  Where possible, we also install the necessary dependencies to use Python 2 as Ansible's discovered Python; unfortunately, this is only possible on Amazon Linux 2, Debian Stretch, Debian Buster, and Ubuntu Bionic.  Note also that no matter the value of this variable, Python 2 will not be installed on Debian Bookworm; this is because the corresponding system packages are unavailable for that platform. | `false` | No |
```

Reply:
I'm a little torn about this because it _should_ be capitalized since it's based on a person's name, yet it is also common to _not_ capitalize it.  This word is very similar to "cartesian" and "abelian" in mathematics in that it is used so much that the capitalization has started to fall away.

That said, it is more correct to capitalize it and therefore I shall!